presto-spi/src/test/java/com/facebook/presto/spi/type/CharsTest.java

```diff
@@ -0,0 +1,109 @@
+/*
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package com.facebook.presto.spi.type;
+
+import io.airlift.slice.Slice;
+import org.testng.annotations.Test;
+
+import static com.facebook.presto.spi.type.Chars.sliceLengthWithTruncateAndTrimTrailingSpaces;
+import static com.facebook.presto.spi.type.Chars.sliceLengthWithoutTrailingSpaces;
+import static io.airlift.slice.Slices.utf8Slice;
+import static io.airlift.slice.Slices.wrappedBuffer;
+import static org.testng.Assert.assertEquals;
+import static org.testng.Assert.fail;
+
+public class CharsTest
```

Comment:
In Presto, we call test classes `TestX`. Therefore, call this `TestChars`. The existing `VarcharsTest` is unfortunately named.